app/assets/stylesheets/components/_navigation-sidebar.scss

```diff
@@ -0,0 +1,38 @@
+.app-c-navigation-sidebar {
+  border-top: 2px solid $govuk-blue;
+}
+
+.app-c-navigation-sidebar__main-heading {
+  margin-top: $gutter-half;
+  margin-bottom: $gutter-one-third;
+  @include bold-19;
+}
+
+.app-c-navigation-sidebar__sub-heading {
```

Comment:
Presumably you're just keeping the existing design here but is it odd that these headings aren't really styled at all? It's strange that they have a smaller font size than the links they're a heading for.

Reply:
This is a change from the existing [related links component](https://govuk-static.herokuapp.com/component-guide/related_items) we have, but it follows @alextea 's design for this component.